packages/roosterjs-editor-plugins/lib/plugins/ImageEdit/ImageEdit.ts

```diff
@@ -329,32 +315,27 @@ export default class ImageEdit implements EditorPlugin {
      * Create editing wrapper for the image
      */
     private createWrapper(operation: ImageEditOperation | CompatibleImageEditOperation) {
-        // Wrap the image with an entity so that we can easily retrieve it later
-        const { wrapper } = insertEntity(
-            this.editor,
-            IMAGE_EDIT_WRAPPER_ENTITY_TYPE,
-            wrap(this.image, KnownCreateElementDataIndex.ImageEditWrapper),
-            false /*isBlock*/,
-            true /*isReadonly*/
-        );
+        //Clone the image and insert the clone in a entity
+        this.clonedImage = this.image.cloneNode(true) as HTMLImageElement;
+        const wrappedImage = wrap(this.clonedImage, KnownCreateElementDataIndex.ImageEditWrapper);
+        this.wrapper = wrap(wrappedImage, KnownCreateElementDataIndex.ImageEditContainer);
 
-        wrapper.style.position = 'relative';
-        wrapper.style.maxWidth = '100%';
         // keep the same vertical align
         const originalVerticalAlign = this.getStylePropertyValue(this.image, 'vertical-align');
         if (originalVerticalAlign) {
-            wrapper.style.verticalAlign = originalVerticalAlign;
+            this.wrapper.style.verticalAlign = originalVerticalAlign;
         }
 
-        wrapper.style.display = Browser.isSafari ? 'inline-block' : 'inline-flex';
+        this.wrapper.style.display = Browser.isSafari ? 'inline-block' : 'inline-flex';
 
         // Cache current src so that we can compare it after edit see if src is changed
         this.lastSrc = this.image.getAttribute('src');
 
         // Set image src to original src to help show editing UI, also it will be used when regenerate image dataURL after editing
         this.image.src = this.editInfo.src;
```

Comment:
I still have concern here. If you need to compare, can you compare with the cloned image?

My primary goal here is not to do any change to editor content when edit.